README.md

```diff
@@ -86,15 +109,12 @@ The names of all views and materialized views are stored in the `metric_views` c
  - `last_run_time`
     - How long the last run of this refresh took
  - `active`
-    - If view_name is a materalized view, determines whether it should be refreshed as part of automatic maintenance
-    - Both matviews and normal views can also set this to be false as a means of allowing external scrape tools to ignore them
+    - Boolean to determine whether this materialized view is refreshed as part of automatic maintenance. Defaults to true. If false, this matview will not be refreshed automatically.
```

Comment:
```suggestion
    - Boolean to determine whether this materialized view is refreshed as part of automatic maintenance. If false, this matview will not be refreshed automatically. Defaults to true. 
```

there are a few places elsewhere that follow this pattern and should be tweaked similarly imho